Project/Src/Gui/FoldMargin.cs

```diff
@@ -110,7 +110,7 @@ bool SelectedFoldingFrom(IEnumerable<FoldMarker> list)
 
                 DrawFoldMarker(
                     g, new RectangleF(
-                        drawingRectangle.X + (drawingRectangle.Width - foldMarkerSize)/2,
+                        drawingRectangle.X + (drawingRectangle.Width - foldMarkerSize)/2f,
```

Comment:
This may have been intentional, to keep pixel rendering crisp on whole pixel boundaries, rather than fuzzy when rendered to subpixels.

Reply:
but e.g. 2f+3f != 5f so rounding should be done on the result 
(sure, this could make a visible difference)

Reply:
It might make sense to do after, but worth checking. As it stands, this could well create fuzzy rendering.

Reply:
This is in "PaintFoldMarker" and GE do not expose that control, so this should not have any effect...